### ISSUE TITLE
Make portal Mobile friendly

### ISSUE THREAD

**Issue by nikeshbalami** [okfnepal/election-nepal#20]
The current site is great but we are already receiving lots of queries regarding site compatibility on mobile devices. What can be done to make it friendly?

**Comment by msucil** [comment]
I think we need to re-structure the page template to make to more compatible with mobile. I'll give a try on this but it takes time.

**Comment by nikeshbalami** [comment]
Yeah, recently came to know about it, because of the Homepage Map visualization it would be difficult to make. Let's give a try 👍

**Comment by msucil** [comment]
I have made changes for other pages except index page. Other pages are mobile friendly and I hope home page will be mobile friendly soon.
